examples/qwik/src/utils/grafbase.ts

```diff
@@ -0,0 +1,18 @@
+export const grafbaseClient = async <T>(body: {
+  query: string
+  variables: Record<string, any>
+}): Promise<T> => {
+  const response = await fetch(
+    import.meta.env.VITE_GRAFBASE_API_URL as string,
+    {
+      method: 'POST',
+      headers: {
+        'content-type': 'application/json',
+        'x-api-key': import.meta.env.VITE_GRAFBASE_API_KEY as string
```

Comment:
The `x-api-key` shouldn't be used on the client.

Is there a way with Qwik to process requests on the server?